src/components/Link/Link.test.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Link from './Link';
+
+const defaultProps = {
+  href: 'https://www.example.com',
+};
+
+it('should match Snapshot', () => {
+  const component = shallow(
+    <Link {...defaultProps} />
+  );
+
+  expect(component.debug()).toMatchSnapshot();
+});
+
+it('should render the children', () => {
+  const component = shallow(
+    <Link {...defaultProps} >
+      Example dot com
+    </Link>
+  );
+
+  expect(component.text()).toBe('Example dot com');
+});
+
+it('should render the href as the text if no children are supplied', () => {
+  const component = shallow(
+    <Link {...defaultProps} />
+  );
+
+  expect(component.text()).toBe('https://www.example.com');
+});
+
+describe('when the external flag is not set', () => {
+  it('should set no target', () => {
+    const component = shallow(
+      <Link  {...defaultProps} />
+    );
+
+    expect(component.prop('target')).toBe(null)
+  });
+
+  it('should add rel noopener noreferrer', () => {
+    const component = shallow(
+      <Link  {...defaultProps} />
+    );
+
+    expect(component.prop('rel')).toBe(null)
+  });
+});
+
+describe('when the external flag is true', () => {
```

Comment:
All use cases appear to be external. Why not make this behaviour the default (or at least, if the href is http(s))? Maybe with an 'internal' attribute to suppress it (though I can't see why we would want to use this component for an internal link).

Reply:
Good point. I have removed all unused complexity (it is always external now). Considered renaming it `<ExternalLink>` but did not.